backend/question-service/src/app.ts

```diff
@@ -1,19 +1,41 @@
 import "dotenv/config";
 import express, { Request, Response, NextFunction } from "express";
 import questionRoutes from "./routes/questions";
+import adminQuestionRoutes from "./routes/adminQuestions";
 import morgan from "morgan";
 import createHttpError, { isHttpError } from "http-errors";
 import cors from "cors";
+import cookieParser from "cookie-parser";
+import * as AuthMiddleWare from "./middleware/authMiddleware";
 
 const app = express();
 
-app.use(cors());
+const FRONTEND_URL = process.env.FRONTEND_URL as string;
+
+// CORS Important Specifications (START)
+app.use(
+  cors({ origin: FRONTEND_URL, optionsSuccessStatus: 200, credentials: true })
+);
+
+app.use((req, res, next) => {
+  res.header('Access-Control-Allow-Origin', FRONTEND_URL);
+  res.header('Access-Control-Allow-Credentials', 'true');
+  res.header('Access-Control-Allow-Headers', 'Origin, X-Requested-With, Content-Type, Accept');
+  next();
+});
```

Comment:
what are these lines for? i tried to comment it out and I still could get/create questions. 🤔

Reply:
Not needed, removed